android/app/src/main/AndroidManifest.xml

```diff
@@ -52,13 +52,20 @@
         </intent>
     </queries>
     <uses-permission android:name="android.permission.INTERNET"/>
+    <uses-permission android:name="android.permission.MANAGE_MEDIA" />
```

Comment:
We shouldn't need `MANAGE_MEDIA` and `ACCESS_MEDIA_LOCATION` permissions?

Reply:
Done